### ISSUE TITLE
HTML end tag missing in Publish Screen

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#848]
**Describe the bug**
HTML end tag missing in Publish Screen

**To Reproduce**

Install "HTML Validator" extension in your browser.
Go to Publish  (http://localhost:9992/cm/app/?view=publish&site=test&mode=edit&name=index.html&path=%2FSites%2Ftest%2F&pathType=page)
Press F12 key to open Chrome DevTools popup.
Go to the HTML Validator tab to see errors. example - Warning: missing </span> before </p>

**Expected behavior**
All start tags must have an end tag.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by ramdev1980** [comment]
Could not reproduced. So closing it.
